### PR TITLE
chore: update security policy

### PR DESCRIPTION
**Pull request by juanpicado**
I'm getting bunch of emails people asking for support and minor things, just wanna be more clear what's the objective of the email.

### CODE REVIEW

**Comment by juanpicado** [comment]
🎉 my first PR at pnpm 🕺

File: SECURITY.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ We kindly ask you to refrain from malicious acts that put our users, the project
 
 ## Reporting a security issue
 
+> Please, do not use the email provided to report issues are not related with security vulnerabilities
```
Comment (DanielRuf) [inline SECURITY.md:23]:
```suggestion
> Please, do not use the provided email address to report issues which are not related to security vulnerabilities
```

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ We kindly ask you to refrain from malicious acts that put our users, the project
 
 ## Reporting a security issue
 
+> Please, do not use the provided email address to report issues which are not related to security vulnerabilities
```
Comment (DanielRuf) [inline SECURITY.md:23]: With and without comma is correct, not sure if we want to emphasize "please" or remove the comma, which adds some small break when you speak it like "please ... do not" while the other is more like "please do not" which is "more" correct.

```suggestion
> Please do not use the provided email address to report issues which are not related to security vulnerabilities
```

**Review by DanielRuf — APPROVED** [review]
LGTM

Not sure if we should leave the comma there, both versions are correct.

File: SECURITY.md (hunk 2 above)
Comment (juanpicado) [reply SECURITY.md:23]: Agree, I actually want to emphasize that. The mailbox is getting noisy.

**Comment by juanpicado** [comment]
Thanks @DanielRuf really appreciated

**Pull request merged**